test/unit/Crawling.HubSpot.Unit.Test/DateTimeConversionTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Globalization;
+using CluedIn.Core.Utilities;
+using Xunit;
+
+namespace Crawling.HubSpot.Unit.Test
+{
+    public class DateTimeConversionTests
+    {
+        [Theory, InlineData(1560992400000)]
+        public void CheckCanConvertFromUnixEpochToDateTime(long epoch)
+        {
+            var date = DateUtilities.EpochRef.AddMilliseconds(epoch);
+            var iso8601 = date.ToString("O");
+
+            DateTimeOffset dummy;
+            Assert.True(DateTimeOffset.TryParse(iso8601, out dummy));
```

Comment:
@BenFidge you are testing the NET framework `DateTimeOffset` here rather than test a function that wraps this conversion logic. See previous comment about inline code duplication.

Reply:
We discussed this, its performing the same tasks as the Core does in the same sequence when converting between the formats